Nodejs/Product/Nodejs/Snippets/1033/Nodejs/initConfig.snippet

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>initConfig</Title>
+      <Description>Code Snippet for intializing a Grunt configuration object</Description>
+      <Shortcut>initConfig</Shortcut>

```

Comment:
On second thought, let's hold off on grunt for now and see if we can just integrate better with the already built-in support in VS (which includes a snippet)
http://webtooling.visualstudio.com/task-runners/grunt/